export/cmake/CMakeLists.txt.export

```diff
@@ -109,8 +109,9 @@ if (REQUIRE_CXX_API AND NOT ${LIBINT_HAS_CXX_API})
   message(FATAL_ERROR "C++ API cannot be built without Eigen3; configure (via CMake) and install Eigen3 and add the install prefix to CMAKE_PREFIX_PATH, or add -DREQUIRE_CXX_API=OFF to the CMake command line if the C++ API is not required")
 endif()
 
-find_package(Boost 1.29)
-if (Boost_FOUND)
+if (NOT USE_BUNDLED_BOOST)
```

Comment:
good idea to "namespace" variables, please replace `USE_BUNDLED_BOOST` with `LIBINT_USE_BUNDLED_BOOST`